Model/ConfigProvider.php

```diff
@@ -108,8 +129,105 @@ public function getInstallments()
                     break;
                 }
             }
+        } else {
```

Comment:
Caso não exista parcelamento o mesmo não deve ser exigido :male_detective: 
Acredito que possa ser uma validação: `caso o parcelamento esteja desabilitado ou habilitado com o número máximo de parcelas definido como 1` não deve ser exigido que o cliente escolha um número de parcelas. O que acha? :thought_balloon:

Reply:
Removendo o parcelamento não faria com que alguns clientes procurassem o campo para tentar fazer a compra parcelada?

Reply:
Acredito que não pois é o comportamento que temos no Magento 1 e WordPres.
É complicado deixar isso de uma maneira que dê fácil manutenção?
Se o esforço for grande podemos postergar e colher feedback dos clientes sobre esse comportamento :wink: